src/cmd/user.go

```diff
@@ -215,6 +215,41 @@ EOF
 	},
 }
 
+var bulkDeleteUsersCmd = &cobra.Command{
+	Use:     "delete users",
+	Aliases: []string{"bulk delete users"},
+	Short:   "Deletes users from a CSV",
+	Long: `Deletes a list of users from a CSV file with the column headers:
+Email`,
+	Example: `
+cat << EOF | opslevel import delete users -f -
```

Comment:
@tozach - so I think the where we want to put this is in the normal `delete users` command but add a `--bulk` or  `--csv`.  Thats a bit more of a surgery so do you mind if I checkout this branch and muck with the code to get that to work?  

I definitely don't want it under the `import` command